chore(blog): add the march summary (#3966)
asyncapi/website: markdown/blog/2025-march-summary.md

date: 2025-03-31T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2025-blog-banner/banner-march.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: Community Builder and Open Source Fanatic!
excerpt: 'AsyncAPI community status and project updates for March 2025'
---

We started the month with exciting news as [AsyncAPI got accepted to the Google Summer of Code](https://summerofcode.withgoogle.com/programs/2025/organizations/asyncapi) (GSoC'25). In the last weeks, the community has seen many enthusiasts join the workspace in anticipation of being accepted to the program. 
However, this excitement brought some challenges as well. Slack and GitHub became overloaded with messages, pull requests, and issues, which overwhelmed our community members and maintainers. 

We are committed to maintaining a welcoming environment for everyone, which is why we have a Code of Conduct and a committee in place to enforce it. But sometimes, too much enthusiasm can become overbearing, resulting in a few members needing to be temporarily banned to ensure a safe space.

> That said, we view this as a valuable lesson, and a big thank you goes out to our Code of Conduct Committee for stepping up and upholding the rules.

Reviewer: ```suggestion
With that being said, we view this as a valuable lesson. A big thank you to our Code of Conduct Committee for stepping up and upholding the rules.
```

As part of preparing the section, we will have an open discussion on the guidelines and rules. Additionally, we have an open issue with the design for implementing the page for the conference website. I encourage everyone to share their thoughts and [review the AsyncAPI Meetups Section discussion](https://github.com/asyncapi/conference-website/discussions/604) and [the AsyncAPI Meetups Page issue](https://github.com/asyncapi/conference-website/issues/644).

## Spec x Tooling

We are excited to welcome a new project, [Neuroglia.AsyncAPI](https://github.com/asyncapi/net-sdk), a .NET SDK for visualizing and interacting with AsyncAPI documents donated by [Charles d'Avernas](https://www.linkedin.com/in/charles-d-avernas-40836a13/). The project also supports a code-first documentation generator, fluent builders for intuitive and expressive API definitions, and fluent validators to ensure compliance and correctness. All the components also support the AsyncAPI v3. [Be sure to try out Neuroglia.AsyncAPI for your documents](https://github.com/asyncapi/net-sdk), especially if you are a .NET developer!

### RAML Schema Parser

> We are looking to archive the RAML schema parser, a standard for RAML data types, unless a community member is interested in maintaining the project. If you have any thoughts on this issue, [please share them in the open discussion about the RAML Parser](https://github.com/asyncapi/raml-dt-schema-parser/issues/255).

Reviewer 2: ```suggestion
At the moment we do not have active maintianers in RAML schema parser, a standard for RAML data types. If there is any community member who depends on this parser and is interested in maintaining the project, reach out to us. If you have any thoughts on this issue, [please share them in the open discussion about the RAML Parser](https://github.com/asyncapi/raml-dt-schema-parser/issues/255).
```

just suggesting different messaging, but probably can be written better than I did

## AsyncAPI Governance Board 
 We are still discussing the new AsyncAPI Governance Board, which will take effect later this year. Part of the changes this will bring will be to replace the single Executive Director position and have the option of Ambassadors becoming TSC members. You still have a chance to share your thoughts and views; please do so [under the introduction of the governance board discussion](https://github.com/asyncapi/community/pull/1634).

## AsyncAPI Bounty Program
[The AsyncAPI Bounty Program Q2 is underway](https://github.com/orgs/asyncapi/projects/36/views/4?sliceBy%5Bvalue%5D=bounty%2F2025-Q2), and we have 17 issues filled into capacity and assigned. It's exciting to see the program grow and the maintainers fully participating.

## TSC x Ambassadors
We are thrilled to welcome three new [Technical Steering Committee](https://www.asyncapi.com/community/tsc) (TSC) members.

<Profiles profiles={[
{

>     name: 'Charles d'Avernas',

Reviewer 3: ```suggestion
    name: "Charles d'Avernas",
```

Author: @akshatnema what's the requested change for?

Author: Nevermind I see why 😅

    avatar: 'https://avatars.githubusercontent.com/u/16137162?v=4',
    link: 'https://github.com/cdavernas'
  }, 
  {
    name: 'Vishvamsinh Vaghela',
    avatar: 'https://avatars.githubusercontent.com/u/90895835?v=4',
    link: 'https://github.com/vishvamsinh28'
  },
  {
    name: 'Jean-Baptiste Bianchi',
    avatar: 'https://avatars.githubusercontent.com/u/72747835?v=4',
    link: 'https://github.com/jbbianchi'
  }
]} />

> And finally, one of our long-time community member.

Reviewer: ```suggestion
And finally, one of our long-time community member:
```
